app/routers.py

```diff
@@ -0,0 +1,24 @@
+import asyncio
+from datetime import datetime
+from fastapi import APIRouter
+from .graphql.queries import Session
+from .config import config
+from .scheduler import build_scheduler
+from .process_manager import ProcessManager as PM
+from .process_manager import TaskType
+router = APIRouter()
+
+
+@router.get("/odb")
+async def odb_changes_endpoint():
+    s = Session(url=config.graphql.url)
+    while True:
+        resp = await s.subscribe_all()
+        if resp:
+            print(resp)
+            # Run new Schedule
+            mode = TaskType.STANDARD
+            scheduler = build_scheduler()
+            PM().add_task(datetime.now(), scheduler, mode)
```

Comment:
This syntax just seems confusing to me... I assume `PM` is a singleton... it seems like we should be getting an instance of `PM` rather than using the create new object syntax.

Reply:
`()` should return the instance based on the metaclass. It removes the need for the `instance()` method.

Reply:
Yeah, that's what I figured. It just seems a bit confusing to me since most people would assume you're creating a new `PM` but we can leave it.